great_expectations/cli/python_subprocess.py

```diff
@@ -106,7 +106,7 @@ def execute_shell_command_with_progress_polling(command: str) -> int:
                 errors=None,
             ) as proc:
                 poll_status_code: Optional[int] = proc.poll()
-                poll_stdout: str = proc.stdout.readline()
+                poll_stdout: str = proc.stdout.readline()  # type: ignore[union-attr] # stdout could be None
```

Comment:
what about `proc.stdout.readline()` or ""`?

Reply:
If `stdout` is `None` we will get an attribute error.